src/ophys_etl/transforms/mesoscope_2p/conversion_utils.py

```diff
@@ -22,12 +23,15 @@ def dump_dict_as_attrs(h5fp, container_name, data):
 
 def volume_to_h5(h5fp, volume, dset_name="data", page_block_size=None,
                  **h5_opts):
+    if isinstance(h5fp, str):
```

Comment:
If `h5fp` is not a string, `f` will not be opened and the `f.create_dataset` calls below will raise an error. Should we be forcing `h5fp` to be a string, or including an `else` block that assignes `f=h5fp` (in the event that `h5fp` is an `h5py` file handle)?

Also, you never explicitly close `f`.

Reply:
@Matyasz I don't think this is resolved.

Reply:
@djkapner whats weird is this doesn't exist anymore when I look at the files changed, but it still exists in this comment :/

Reply:
Oh when I look at this comment from the conversation tab it says this comment is `outdated`. It didn't show that when I looked at the files changed tab though, huh

Reply:
I totally removed this `str` capability from this method, as discussed in another comment.

Reply:
I made another comment elsewhere where I saw a file not closed. I'll resolve this conversation and follow up in that thread.